src/components/CreatePost/StepperButton.tsx

```diff
@@ -32,17 +39,16 @@ const StepperButton = ({ format, steps, setSteps, onClick }: StepperButtonProps)
           onClick={() => {
             if (onClick) {
               onClick(onSubmit)();
```

Comment:
proposal

```suggestion
          onClick={() => {
            onClick && onClick(onSubmit)();
```

조건문 대신 논리 연산자을 사용하면 불필요한 조건문을 제거할 수 있다고 생각합니다.